### PR TITLE
Javascript/javascript2/week3

### PR DESCRIPTION
**Pull request by SlaviaF**

### CODE REVIEW

File: javascript/javascript2/week3/function.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+//REGULAR FUNCTION
+//with regular function it just logs out the functon name
+function house() { console.log('apartment', 'villa') };
+function road() { console.log('cobbled', 'cemented') };
+function furniture() { console.log('sofa', 'chairs') };
+
+const newFunctionArray = [house(), road(), furniture()];
+
+newFunctionArray.forEach(item => item);
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/function.js]: Your array newFunctionArray will end up equalling [undefined, undefined, undefined] 
```suggestion
const newFunctionArray = [house, road, furniture];

newFunctionArray.forEach(item => item());
```

File: javascript/javascript2/week3/game.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        if (winnerObj.sPlayer > winnerObj.lPlayer) {
+            win.innerHTML = "Congrats Player S";
+
+        }
+        if (winnerObj.sPlayer < winnerObj.lPlayer) {
+            win.innerHTML = "Congrats Player L";
+        }
+        if (winnerObj.sPlayer == winnerObj.lPlayer) {
+            win.innerHTML = "It's a draw, Try Again";
+        }
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/game.js]: Use some blank lines to separate these if statements.
```suggestion
        if (keyS > keyL) {
            playerOnLead.innerHTML = `Player S on lead`
        }
        if (keyS < keyL) {
            playerOnLead.innerHTML = `Player L on lead`
        }
        //
        if (winnerObj.sPlayer) {
            clickCountS.innerText = winnerObj.sPlayer;
        }
        else if (winnerObj.lPlayer) {
            clickCountL.innerText = winnerObj.lPlayer;
        }

        if (winnerObj.sPlayer > winnerObj.lPlayer) {
            win.innerHTML = "Congrats Player S";
        }
        else if (winnerObj.sPlayer < winnerObj.lPlayer) {
            win.innerHTML = "Congrats Player L";
        }
       else {
            win.innerHTML = "It's a draw, Try Again";
        }
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week3/function.js (hunk 1 above)
Comment (SlaviaF) [reply javascript/javascript2/week3/function.js]: oohhh yesss...I will change it

File: javascript/javascript2/week3/game.js (hunk 2 above)
Comment (SlaviaF) [reply javascript/javascript2/week3/game.js]: Thanks a lot Cameron..will change it right away

**Pull request merged**